12-hooks-in-react/src/components/Accordion.js

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+
+const Accordion = ({ items }) => {
+  const renderedItems = items.map((item) => {
+    return (
+      <React.Fragment key={item.title}>
```

Comment:
Title is not a good key as if 2 or more titles are the same this'll error out, used the index of the items instead or make your own counter for this that's unique.

Reply:
Implemented the change of key prop to index. Please check my latest commit.